documentation/concepts/check.md

```diff
@@ -73,6 +73,29 @@ Flyway’s `check –changes` will then:
 1. Compare the V2 build database snapshot to the V5 build database snapshot
 1. Generate a HTML (human readable) and JSON (machine readable) Change Report, indicating the additions, deletions, and modifications of database objects between V2 and V5
 
+#### Example: `appliedMigrations` and `buildUrl`
+
+The `check –changes` command and flag works by building a temporary database. This 'build' database is first made to reflect the state specified by `appliedMigrations`, and then made to reflect your `appliedMigrations` with the pending changes applied.
+
+The difference between the two states of this build database (`appliedMigrations`, and `appliedMigrations` with changes applied) represents the effect your pending migrations will have (or have had) when the scripts are (or were) executed. This difference is captured as an artefact called a “Change Report”. The change report is available as both HTML (human readable) and JSON (machine readable) formats.
+
+The process works like this:
+![Check_appliedMigrations.png](/assets/balsamiq/Check_appliedMigrations.png)
+1. Run `flyway info -infoOfState="success,pending,out_of_order" -migrationIds > appliedMigrations.txt`
+    1. This will produce a comma-separated list which represents the applied migrations of your target database
+1. Specify a build database
```

Comment:
Does this step need to be more specific? How do users specify the build database?